src/foam/nanos/crunch/connection/CapabilityPayloadDAO.js

```diff
@@ -259,7 +259,7 @@ foam.CLASS({
 
         List leaves = new ArrayList<>();
         CrunchService crunchService = (CrunchService) x.get("crunchService");
-        List grantPath = crunchService.retrieveCapabilityPath(x, receivingCapPayload.getId(), true, true, leaves);
+        List grantPath = crunchService.retrieveCapabilityPath(x, receivingCapPayload.getId(), false, true, leaves);
```

Comment:
this update will make granted capable capabilities, updatable.

When/Why does this happen?
Generally with UCJ's if the data is granted we don't allow updates atm cause it can cascade into other actions. We will be implementing edits on granted capabilities but will do so carefully with a history of these changes.

Reply:
@Nauna Yeah making granted capabilities updatable was the intention of the change. In intuit, there are times when we simply want to update the existing onboarding capability if a user has onboarded and tries to onboard under the same flinks login id again. This prevents creating duplicate users in our system. Is the implementation ongoing? Without this feature, onboarding won't work as expected in some cases. @nanoscott